DESCRIPTION

```diff
@@ -27,12 +27,12 @@ Imports:
     magrittr,
     memoise,
     methods,
+    openssl,
```

Comment:
I'd rather not import _openssl_ . _tic_ should aim for a small footprint, we even might be able to move _git2r_ and _base64enc_ to `Suggests`.

Reply:
I did so.